README.md

```diff
@@ -1,65 +1,52 @@
-    * Call the Star Wars API and print results
-    * Store API results in React component state
-    * Render React state
-* A plan for the next 3 weeks and what you plan to accomplish each week
-
-## Recommendations
-
-* Keep the scope of your project small. A small, completed project is much better than a large, incomplete project. You can create a list of "nice to have" features that you can work on if you have time (similar to how our assignments have a main list of requirements and "bonus" items).
-* Ensure you have plenty of buffer time in your schedule. Life events pop up, and tasks can take longer than initially estimated.
-
-## Submitting your Project Idea
-
-1. Fork this repository
-2. Add your project name, elevator pitch, wireframes, dependencies, task list, and plan to the Github repo (you can add a markdown file + images or links to images)
-3. Create a pull request
+## Elevator Pitch
+ For the final project, I would like to create a dictionary word search and word memorization application - This application allows users to search words from the owlbot dictionary api, then add select words to their collection to review later. Words in the users collection can be reviewed as flashcards or as dynamically generated tests. This a handy application for individuals to both look up the meanings of words and then review those words later to improve their vocabulary.
+
+The main inspiration for this application comes from existing websites like dictionary.com and quizlet.com. Quizlet.com is application that allows you to create your own flashcards and quizzes. I'm hoping to make a simplified version of this by using pre-existing data from the owlbot api and focusing specifically on words rather than other types of media which is allowed on quizlet.com
+
+While brainstorming I was focused on finding a project that would allow me to integrate all of the topics we have discussed including react-router, fetch api, firebase authentication, and firebase firestore.
+
+## Wireframes and Design Concept
+My design concept for this project is a minimalist approach with lots of whitespace and some pops of color with the patterns found [here](https://products.ls.graphics/paaatterns/) as inspiration. As a stretch goal I would love to set up a themeing functionality in which the user can change the theme and switch between a light and dark mode. Mobile responsiveness will be in consideration throughout the process of design and implementation as well.
+
+Here are some wireframes for the homepage, search results, and users' collection page. These are not all the view of the webpage, but a starting point to begin working on my final project.
+
+![Wireframe](/images/Views.png) 
+
+
+
+## Dependencies
+  * React
+  * Firebase Authentication
+  * Firebase Firestore
+  * [Owlbot API](https://owlbot.info/) (for dictionary queries)
+  * [Wordnik API](https://developer.wordnik.com/) (to get "word of the day" functionality - still waiting to obtain the api key)
```

Comment:
I would make sure you have a backup for this (sometimes API services are notorious for taking a while/being unsupported). Also, you _shouldn't_ expose this API key in the React app. It should be hidden behind a web service of some sort, particularly if you're using a paid subscription. If you're on the free tier, it doesn't matter as much.